src/fhirtypes/StructureDefinition.ts

```diff
@@ -263,8 +328,11 @@ export class StructureDefinition {
    * @param {any} json - the FHIR 3.0.1 JSON representation of a StructureDefinition to construct
    * @returns {StructureDefinition} a new StructureDefinition instance representing the passed in JSON
    */
-  static fromJSON(json: LooseStructDefJSON): StructureDefinition {
-    const sd = new StructureDefinition();
+  static fromJSON(
+    json: LooseStructDefJSON,
+    structureDefinitionStructureDefinition: StructureDefinition = null
```

Comment:
If we take the SD out of the constructor, then we don't need it here anymore either.